client/src/features/sessionsV2/SessionView/SessionStartLinkModal.tsx

```diff
@@ -20,24 +20,25 @@ import cx from "classnames";
 import { Link45deg, XLg } from "react-bootstrap-icons";
 import { Button, Modal, ModalBody, ModalFooter, ModalHeader } from "reactstrap";
 import { CommandCopy } from "../../../components/commandCopy/CommandCopy";
-import { Project } from "../../projectsV2/api/projectV2.api";
-import { SessionLauncher } from "../api/sessionLaunchersV2.generated-api";
+import { Ulid } from "../api/sessionLaunchersV2.generated-api";
 import useSessionStartLink from "./useSessionStartLink.hook";
 
 interface SessionStartLinkModalProps {
   isOpen: boolean;
-  launcher: SessionLauncher;
-  project: Project;
+  launcherId: Ulid;
```

Comment:
This is just a string for the client:

```suggestion
  launcherId: string;
```